cv32e40x/env/corev-dv/cv32e40x_csr_template.yaml

```diff
@@ -365,6 +365,79 @@
       reset_val: 1
       msb: 0
       lsb: 0
+
+### Not supported in CV32E40X ###
```

Comment:
Is all of this commented-out code autogenerated?
Where does it come from?

Reply:
It is _not_ autogenerated, I wrote this manually. The reason why I kept the non-existent registers present, albeit commented out,  was to ease integration with other cores that could potentially use these.